test/Microsoft.AspNet.Mvc.Core.Test/VirtualFileResultTest.cs

```diff
@@ -241,73 +267,29 @@ public void Constructor_SetsContentTypeAndParameters()
             Assert.Equal(expectedData, contents);
         }
 
-        [Theory]
```

Comment:
The tests were meant for behavior that `FileResult` used to support - https://github.com/aspnet/Mvc/commit/ddc74e5396c5a09e950c5d0029bc23d452bc7783#diff-79ab8ced19b9f57a37b11093d8533ed4L112 to detect if a path was rooted . That code's been removed and we pass the path as is (with the exception of removing the tilde).